packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -108,22 +131,26 @@ export class ServerDeploymentGroup extends ServerDeploymentGroupRef {
     public readonly deploymentGroupName: string;
 
     constructor(parent: cdk.Construct, id: string, props?: ServerDeploymentGroupProps) {
```

Comment:
Tip, use `{ }` as the default value for `props`:

```ts
constructor(parent: cdk.Construct, id: string, props: ServerDeploymentGroupProps = { })
```

Then, you can eliminate line 136

Reply:
Good call, changed!